functions.php

```diff
@@ -114,6 +119,29 @@ function sortByDate($a, $b) {
 }
 
 function sanitize($string) {
+	if(strpos($string, "plugin/") === 0){
```

Comment:
The function need to work in plugins and themes, there is some constant to identify if a plugin or theme. The idea is create función they work in both place homepages.

Reply:
Oops, that is my internal function. I guess this isn't needed on your environment. I had the problem, that my .htaccess didn't route the URLs correctly to the plugins and since I hate the Apache configuration, I just adapted the script. :D

I'll remove that.